x-pack/plugins/fleet/server/services/epm/packages/bulk_install_packages.ts

```diff
@@ -107,12 +149,29 @@ export async function bulkInstallPackages({
         return { name: packageName, error: result.reason };
       }
 
-      const pkgKeyProps = result.value;
+      if (result.value.forbidden) {
+        return {
+          name: packageName,
+          status: 'not_installed',
+          error: new Error(
+            `Bulk installation of ${packageName} is forbidden. Please install it via the single package installation API.`
```

Comment:
I think that this generic `Error` will default to a 500. I think it should be something else, probably a `BadRequest` instead. It can be added [here](https://github.com/elastic/kibana/blob/ab21d4fee4f5e9149b85b1a0ce7fc0dbcfaa5062/x-pack/plugins/fleet/server/errors/handlers.ts#L60) and handled [here](https://github.com/elastic/kibana/blob/ab21d4fee4f5e9149b85b1a0ce7fc0dbcfaa5062/x-pack/plugins/fleet/server/errors/handlers.ts#L60).

Reply:
Thanks! Fixed in https://github.com/elastic/kibana/pull/186330/commits/67e814a920f28b56e4abdcc7359cec6a056cd7b4 so that `statusCode: 422` is included with this. 422 felt appropriate as the request "correct" but Fleet can't fulfill the install request for internal reasons. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422.